src/main/java/com/github/black0nion/blackonionbot/api/sessions/WebSocketSession.java

```diff
@@ -9,9 +9,9 @@
 import java.util.InputMismatchException;
 import java.util.concurrent.ExecutionException;
 
-public class BlackWebsocketSession extends BlackSession implements org.eclipse.jetty.websocket.api.Session {
+public non-sealed class WebSocketSession extends GenericSession implements Session {
```

Comment:
Why not use https://github.com/TakahikoKawasaki/nv-websocket-client

Reply:
Again, this is for the server side of things and it's just an utility class for logic like discord sessino handling. Thanks for reminding me that I have to write docs!

Reply:
no problem/